voluncheer/jobboard/models.py

```diff
@@ -2,6 +2,7 @@
 
 
 class Job(models.Model):
+    id=models.IntegerField(primary_key=True)
```

Comment:
Primary keys are added automatically: https://docs.djangoproject.com/en/4.1/topics/db/models/#automatic-primary-key-fields

Is there a reason we need to override it here?